firefox-ios/Client/Frontend/Home/PocketFooterView.swift

```diff
@@ -28,15 +29,16 @@ class PocketFooterView: UICollectionReusableView, ReusableCell, ThemeApplicable
         label.adjustsFontForContentSizeCategory = true
         label.font = FXFontStyles.Regular.caption1.scaledFont()
     }
-
-    private let learnMoreLabel: UILabel = .build { label in
-        label.text = .FirefoxHomepage.Pocket.Footer.LearnMore
-        label.isUserInteractionEnabled = true
-        label.adjustsFontForContentSizeCategory = true
-        label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.Pocket.footerLearnMoreLabel
-        label.font = FXFontStyles.Regular.caption1.scaledFont()
+    
+    private lazy var learnMoreButton: LinkButton = .build { button in
+        button.setTitle(.FirefoxHomepage.Pocket.Footer.LearnMore, for: .normal)
```

Comment:
The button title, a11y id and font should be set through `configure(viewModel:)` of the LinkButton.